chapter-1-basics.md

```diff
@@ -31,7 +31,7 @@ using the provided name and optionally takes additional parameters like `--steps
 command:
 
 ```
-$ moryx new <NAME> --steps <LIST-OF-STEPS> --products <LIST-OF-PRODUCTS>
+$ moryx new <NAME> --steps <LIST-OF-STEPS> --products <LIST-OF-PRODUCTS> --template-url <URL> --branch <NAME>
```

Comment:
I'd say, there is no need to list *all* the parameters here. If we need to also provide a custom template, that is more relevant to be pointed out, where it is needed.